.github/workflows/ci.yml

```diff
@@ -259,49 +261,59 @@ jobs:
           yarn lerna:init
           yarn check:clean-workspace-after-install
 
-      - name: Build
+      - name: Use eslint v${{ matrix.eslint-version }}
         run: |
-          yarn build
+          yarn add -DW eslint@${{ matrix.eslint-version }} --ignore-scripts
+
+      - name: Build
+        run: yarn build
 
       - name: Run unit tests for typescript-estree
         run: yarn test
         working-directory: packages/typescript-estree
+        if: ${{ always() }}
```

Comment:
what is this line for?

Reply:
The idea is that it will continue to run the other unit tests even if one of the unit test steps fail. This is useful to see if unit tests for other packages would also fail instead of early termination of the test run after the first failing unit test. For example, multiple different packages might have unit tests which fail on a specific eslint version and the test runner wouldn't show this as it would skil the other unit tests on the first failing step.

Reply:
I can remove it if you don't think it's a good idea.